libs/cohere/README.md

```diff
@@ -12,6 +12,21 @@ pip install langchain-cohere
 
 - Get a [Cohere API key](https://cohere.com/) and set it as an environment variable (`COHERE_API_KEY`)
 
+## Migration from langchain-community
+
+Cohere's integrations used to be part of the `langchain-community` package, but since version the integration in `langchain-community` has been deprecated in favour `langchain-cohere`.
+
+The two steps to migrate are:
+
+1) Import from langchain_cohere instead of langchain_community, for example:
```

Comment:
```suggestion
1) Import from `langchain_cohere` instead of `langchain_community`, for example:
```

Reply:
I considered this, but it made the section hard to read as there were so many quoted strings! I'd be happy to add your suggestion if you feel strongly about it, though.

Reply:
no worries